backend/src/main/java/com/google/sps/FindMatchQuery.java

```diff
@@ -66,11 +66,10 @@ public Match findMatch(List<Participant> participants, Participant newParticipan
           dateTime.plusMinutes(duration + PADDING_TIME).isBefore(earliestEndTimeAvailable);
 
       if (compatibleTime) {
-        // TODO: change match ID (currently -1 for easy error checking)
```

Comment:
Match IDs can't be assigned, they correspond to the key of the item in the datastore.

Reply:
Does it mean we can't assign a match ID when creating a Match? If so, we should remove this field from the constructor and set the field to a default value. (Otherwise people who's using this constructor may think they have to assign a value for it)

Reply:
Yes, we can't assign a match ID when creating a match. But by saving the match id, given a match, we can find the corresponding match in the datastore. Looking at the rest of the code, it doesn't look like I ever really need to do that, so I can take the matchId out of a Match object.

Reply:
sgtm